scout/constants/indexes.py

```diff
@@ -100,15 +100,16 @@
                 ("case_id", ASCENDING),
                 ("variant_type", ASCENDING),
                 ("rank_score", ASCENDING),
+                ("hgnc_ids", ASCENDING),
```

Comment:
This is the actual speedup, allowing to use this index with hgnc_id filtering, with score for sorting.

Reply:
Nice! 🚀